x-pack/plugins/ml/public/explorer/explorer_utils.js

```diff
@@ -191,6 +191,7 @@ export function getSelectionTimeRange(selectedCells, interval, bounds) {
 export function getSelectionInfluencers(selectedCells, fieldName) {
   if (
     selectedCells !== null &&
+    selectedCells.type !== 'overall' &&
```

Comment:
This could be changed to check again the enum `SWIMLANE_TYPE.OVERALL` from `explorer_constants.js`.